### ISSUE TITLE
potential unconsistency in fuel-types

### ISSUE THREAD

**Issue by sandrinecharousset** [openENTRANCE/nomenclature#112]
the following lines are present in fuel-types.yaml but they do not look like fuel types. I would have removed them and included somewhere else? 

 Electricity:
    description: Energy generation from electricity
   
  Electricity|Heatpump:
    description: Energy generation from Heatpumps
   
  Electricity|Other:
    description: Energy generation from other electrification technologies
    
  Other:
    description: Other energy sources and fuels not explicitely listed

**Comment by danielhuppmann** [comment]
Good point - it would make sense to go carefully through the list where the tag is used and see if this even makes sense - and then maybe create an additional tag-list (e.g., energy-carrier) that has actual fuels as well as electricity...
